builtin/credential/okta/path_config.go

```diff
@@ -153,22 +158,18 @@ func (b *backend) pathConfigWrite(
 		return logical.ErrorResponse("api_token is missing"), nil
 	}
 
-	baseURL, ok := d.GetOk("base_url")
-	if ok {
-		baseURLString := baseURL.(string)
-		if len(baseURLString) != 0 {
-			_, err = url.Parse(baseURLString)
-			if err != nil {
-				return logical.ErrorResponse(fmt.Sprintf("Error parsing given base_url: %s", err)), nil
-			}
-			cfg.BaseURL = baseURLString
-		}
-	} else if req.Operation == logical.CreateOperation {
-		cfg.BaseURL = d.Get("base_url").(string)
+	baseURL := d.Get("base_url").(string)
```

Comment:
This code now requires base_url set at all times instead of honoring create/update. Not necessarily a problem, but a change in behavior. Also, the documentation lists it as optional.

Reply:
I guess this is the problem with providing a default value in the fields.  I'll remove the default and only populate it when it is specified.